operator/v1/types_network.go

```diff
@@ -339,6 +345,40 @@ type HybridOverlayConfig struct {
 type IPsecConfig struct {
 }
 
+type ExportNetworkFlows struct {
+	// netFlow defines the NetFlow configuration.
+	// +optional
+	NetFlow *NetFlowConfig `json:"netFlow,omitempty"`
+	// sFlow defines the SFlow configuration.
+	// +optional
+	SFlow *SFlowConfig `json:"sFlow,omitempty"`
+	// ipfix defines IPFIX configuration.
+	// +optional
+	IPFIX *IPFIXConfig `json:"ipfix,omitempty"`
+}
+
+type NetFlowConfig struct {
+	// netFlow defines the NetFlow collectors that will consume the flow data exported from OVS.
+	// It is a list of strings formatted as ip:port
+	// +kubebuilder:validation:MinItems=1
+	NetFlowCollectors []IPPort `json:"netFlowCollectors,omitempty"`
```

Comment:
Just `Collectors`/`collectors`. It's already in a subfield called `netFlow`, you don't need to say that again. (Likewise with the others.)